.gitignore

```diff
@@ -11,7 +11,7 @@ tmp*.*
 
 _site
 .coverage
-/node_modules
+**/node_modules
```

Comment:
there should not be an node_modules in the packages , do you see any ?

Reply:
This is needed in the case that a user adds a new dependency that conflicts with the global workspace dependencies. In this case, it will add a `./node_modules` folder within the scope of the package. This prevents those from being committed.